docs/_posts/2015-09-14-community-roundup-27.md

```diff
@@ -0,0 +1,78 @@
+---
+title: "Community Round-up #27 &ndash; Relay Edition"
+layout: post
+author: steveluscher
+---
+
+In the weeks following the [open-source release](/react/blog/2015/08/11/relay-technical-preview.html) of the Relay technical preview, the community has been abuzz with activity. Let's take a look at some of the things we've achieved, together!
+
+## Teaching servers to speak GraphQL
+
+Every great Relay app starts by finding a GraphQL server to talk to. The community has spent the past few weeks teaching GraphQL to a few backend systems.
+
+Bryan Goldstein [brysgo](https://github.com/brysgo) has built a tool to help you define a GraphQL schema that wraps a set of [Bookshelf.JS](http://bookshelfjs.org/) models. Check out [graphql-bookshelf](https://github.com/brysgo/graphql-bookshelf).
+
+RisingStack ([risingstack](https://github.com/risingstack)) created a GraphQL ORM called [graffiti](https://github.com/RisingStack/graffiti) that you can plug into [mongoose](http://mongoosejs.com/), and serve using Express, Hapi, or Koa.
+
+David Mongeau-Petitpas ([dmongeau](https://github.com/dmongeau)) is working on a way to vend your Laravel models through a GraphQL endpoint, [laravel-graphql](https://github.com/Folkloreatelier/laravel-graphql).
+
+Gerald Monaco ([devknoll](https://github.com/devknoll)) created [graphql-schema](https://github.com/devknoll/graphql-schema) to allow the creation of JavaScript GraphQL schemas using a fluent/chainable interface.
+
+Jason Dusek ([solidsnack](https://github.com/solidsnack)) dove deep into PostgresQL to teach it how to respond to GraphQL query strings as though they were SQL queries. Check out [GraphpostgresQL](https://github.com/solidsnack/GraphpostgresQL).
+
+Espen Hovlandsdal ([rexxars](https://github.com/rexxars)) built an [sql-to-graphql](https://github.com/vaffel/sql-to-graphql) tool that can perform introspection on the tables of a MySQL or PostgresQL database, and produce a queryable HTTP GraphQL endpoint out of it.
+
+Mick Hansen ([mickhansen](https://github.com/mickhansen)) offers a set of [schema-building helpers](https://github.com/mickhansen/graphql-sequelize) for use with the [Sequelize ORM](http://docs.sequelizejs.com/en/latest/) for MySQL, PostgresQL, SQLite, and MSSQL.
+
+## GraphQL beyond JavaScript
+
+Robert Mosolgo ([rmosolgo](https://github.com/rmosolgo)) brought the full set of schema building and query execution tools to Ruby, in the form of [graphql-ruby](https://github.com/rmosolgo/graphql-ruby), [graphql-relay-ruby](https://github.com/rmosolgo/graphql-relay-ruby). Check out his [Rails-based demo](https://github.com/rmosolgo/graphql-ruby-demo).
```

Comment:
"schema building" → "schema-building"

And I think you want an "and" instead of a comma between the graphql-ruby and graphql-relay-ruby links.